pkg/api/http_server.go

```diff
@@ -5,6 +5,7 @@ import (
 	"crypto/tls"
 	"errors"
 	"fmt"
+	"github.com/grafana/grafana/pkg/services/provisioning"
```

Comment:
This insertion seems weird. It should be placed with all other local imports at line 15